### PR TITLE
libview 0.6.6 (new formula)

### PR DESCRIPTION
**Pull request by vicamo**
Signed-off-by: You-Sheng Yang (楊有勝) <vicamo@gmail.com>

### CODE REVIEW

File: Library/Formula/libview.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+class Libview < Formula
+  desc "VMware's incredibly exciting widgets"
```
Comment (bfontaine) [inline Library/Formula/libview.rb]: Could we be a little more specific than that?

File: Library/Formula/libview.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  sha256 "394583d2d28c334684d512eb2a260db51a877b84319f74a2425b79b5780f3ad8"
+  head "svn://svn.code.sf.net/p/view/code/trunk/libview"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gtk+"
+  depends_on "gtkmm"
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on :x11
```
Comment (bfontaine) [inline Library/Formula/libview.rb]: If it has a hard dependency on X11 it’ll need to be submitted to our [`homebrew/x11`](https://github.com/Homebrew/homebrew-x11) tap.

File: Library/Formula/libview.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  # upstream bug: http://sourceforge.net/p/view/bugs/1/
+  patch do
+    url "http://sourceforge.net/p/view/bugs/_discuss/thread/7b22d179/1941/attachment/libview-0.5.6.diff"
+    sha256 "4d93c8067fb1d371780a665fff731dd27ce3daac983ea63efbb7ba2bad492d39"
+  end
+
+  def install
+    # Homebrew renames libtool to glibtool intentionally.
+    inreplace "autogen.sh", "libtool", "glibtool"
```
Comment (bfontaine) [inline Library/Formula/libview.rb]: You might want to use a regexp to ensure we don’t replace `glibtool` with `gglibtool`.

File: Library/Formula/libview.rb (hunk 1 above)
Comment (vicamo) [inline Library/Formula/libview.rb]: That's from libview's homepage.

File: Library/Formula/libview.rb (hunk 2 above)
Comment (vicamo) [inline Library/Formula/libview.rb]: Will do.

**Comment by vicamo** [comment]
Re-submitted as https://github.com/Homebrew/homebrew-x11/pull/183

**Pull request closed without merging**